evap/staff/importers.py

```diff
@@ -303,8 +306,26 @@ def write_enrollments_to_db(self, semester, vote_start_date, vote_end_date):
                 student = UserProfile.objects.get(email=student_data.email)
                 course.participants.add(student)
 
-        self.success_messages.append(_("Successfully created {} course(s), {} student(s) and {} contributor(s).").format(
-            len(self.courses), students_created, responsibles_created))
+        msg = _("Successfully created {} course(s), {} student(s) and {} contributor(s):").format(
+            len(self.courses), len(students_created), len(responsibles_created))
+        for user in students_created + responsibles_created:
+            msg += "<br>"
+            msg += "{} {} ({})".format(user.first_name, user.last_name, user.username)
+        self.success_messages.append(mark_safe(msg))
```

Comment:
any specific reason why this is not in an extra method? i think it's a bit confusing that the test run has a separate method, but this here does not